README.md

```diff
@@ -1,16 +1,275 @@
 # django-dequorum
-A simple web forum for Django
+A simple web forum Django application
+
+# Environment setup
+python3 environment setup. You can use *either* virtualenv ([documentation](http://docs.python-guide.org/en/latest/dev/virtualenvs/)) *or* virtualenv with virtualenvwrapper ([documentation](https://virtualenvwrapper.readthedocs.org/en/latest/index.html)) installed.
+
+* [virtualenv](https://pypi.python.org/pypi/virtualenv)
+
+    Create new virtual environments directory, e.g. `venvs`
+    ```
+    ~$ mkdir venvs
+    ~$ cd venvs/
+    ~/venvs$ 
+    ```
+
+    Create new virtual environment
+    ```
+    ~/venvs$ virtualenv -p `which python3` melb-django
+    ```
+
+    To start working on `django-python3` virtualenv
+    ```
+    ~/venvs$ cd melb-django/
+    ~/venvs/melb-django$ source bin/activate
+    ```
+ 
+    To stop working on `django-python3` virtualenv
+
+    ```
+    (django-python3) ~/venvs/melb-django$ deactivate
+    ```
+ 
+* virtualenv with [virtualenvwrapper](https://pypi.python.org/pypi/virtualenvwrapper/) installed
+
+    ``` 
+    ~$ mkvirtualenv -p `which python3` django-python3
+    ```
+
+    To start working on `django-python3` virtualenv
+
+    ```
+    ~$ workon django-python3
+    (django-python3) ~$
+    ```
+    
+    To stop working on `django-python3` virtualenv
+
+    ```
+    (django-python3) ~$ deactivate
+    ```
+
+Notes: 
+
+* `(django-python3)` prompt change at your terminal indicating you're on `django-python3` virtualenv.
 
 # Installation
```

Comment:
Perhaps rename this to "Quickstart", since this isn't the general install procedure.